managing_providers/cloud_providers/cic_providers.md

```diff
@@ -0,0 +1,16 @@
+---
+---
+
+# IBM Cloud Infrastructure Center (CIC) Providers
+
+## IBM Cloud Infrastructure Center (CIC)
+
+IBM Cloud Infrastructure Center (CIC) is an advanced infrastructure management (OpenStack compatible) software that includes on-premises deployments of IBM z/VM®- and KVM-based Linux® virtual machines on the IBM Z® and LinuxONE platforms.
+
+## Adding IBM CIC Providers
+
+{% include openstack/adding-a-cic-cloud-provider.md %}
```

Comment:
Following the PowerVC style:

```suggestion
{% include_relative _topics/adding_ibm_cic_providers.md %}
```

The top level includes is typically for things which are shared across many pages where as this is not.